uxarray/grid/grid.py

```diff
@@ -231,6 +231,24 @@ def from_dataset(
 
         return cls(grid_ds, source_grid_spec, source_dims_dict)
 
+    @classmethod
+    def from_geodataframe(
+        cls,
+        filename: str,
+        **kwargs,
+    ):
+        """Constructs a ``Grid`` object from a shapefile.
```

Comment:
This should be updated.

Reply:
thanks for the catch, fixed. please review.